spacy/gold/example.pyx

```diff
@@ -15,7 +15,7 @@ from ..syntax import nonproj
 
 
 cpdef Doc annotations2doc(vocab, tok_annot, doc_annot):
-    """ Create a Doc from dictionaries with token and doc annotations. Assumes ORTH & SPACY are set. """
+    """ Create a Doc from dictionaries with token and doc annotations. """
     attrs, array = _annot2array(vocab, tok_annot, doc_annot)
     output = Doc(vocab, words=tok_annot["ORTH"], spaces=tok_annot["SPACY"])
```

Comment:
The assumption of ORTH and SPACY being set is still present in this line though ...

Reply:
The `SPACY` value can be `None` by `_guess_spaces`, so it felt weird to say "are set". I guess we could clarify?

Reply:
Right, with "set" I meant "they are in the dictionary". I think it makes sense to expect that before this function is called, but like you said the value can still be `None`, which I think works fine further downstream.